src/strings.c

```diff
@@ -1056,6 +1056,7 @@ const u8 gText_BerryCrush2[] = _("BERRY CRUSH");
 const u8 gText_PressingSpeedRankings[] = _("Pressing-Speed Rankings");
 const u8 gText_Var1Players[] = _("{STR_VAR_1} PLAYERS");
 ALIGNED(4) const u8 gText_Ghost[] = _("GHOST");
+ALIGNED(4) const u8 gText_Luvdisc[] = _("LUVDISC");
```

Comment:
Is this variable necessary?
Outside of a commented out line I don't see it used anywhere.

Reply:
good catch. This one is actually no longer required. Was part of an earlier implementation approach.